FastSimulation/SimplifiedGeometryPropagator/plugins/FastTrackDeDxProducer.cc

```diff
@@ -0,0 +1,218 @@
+// -*- C++ -*-
+//
+// Package:    FastTrackDeDxProducer
+// Class:      FastTrackDeDxProducer
+// 
+/**\class FastTrackDeDxProducer FastTrackDeDxProducer.cc RecoTracker/FastTrackDeDxProducer/src/FastTrackDeDxProducer.cc
+
+   Description: <one line class summary>
+
+   Implementation:
+   <Notes on implementation>
+*/
+//
+// Original Author:  andrea
+//         Created:  Thu May 31 14:09:02 CEST 2007
+//    Code Updates:  loic Quertenmont (querten)
+//         Created:  Thu May 10 14:09:02 CEST 2008
+//
+//
+
+
+// system include files
+//#include "RecoTracker/DeDx/plugins/DeDxEstimatorProducer.h"
```

Comment:
remove the commented out line if not needed

Reply:
removed